src/block-experiments-setup.js

```diff
@@ -0,0 +1,5 @@
+// This file is to set up the jetpack/layout-grid block that currently lives in block-experiments/blocks/layout-grid
+
+import { registerBlock } from '../block-experiments/blocks/layout-grid/src';
```

Comment:
I just had a thought about the relative import path used here. Not suggesting a change, but wondering if there's a way to scope this (similar to how we resolve `@wordpress` dependencies) such that a future change in project structure wouldn't require any changes here. Something like `'@something/blocks'` :thinking: 

If there's too much "magic" in the module resolution configuration, perhaps we'd still achieve a similar effect by importing from an index at a higher level in `../block-experiments`? It feels like "reaching deep into" the path here will be brittle to structure changes in the dependency. (This comment might belong there as well as here :sweat_smile: ) Wdyt?

Reply:
I think that is a great idea but I am not sure how to go about doing that. Any suggestions?

Reply:
The second idea would just involve creating an `index.js` (or possibly `index.native.js`) file in `../block-experiments/blocks/` and re-exporting the block registration there. This would shift the responsibility of maintaining the deeper path traversals to the dependency, which probably makes more sense, since that's where the structure changes would occur.

Regarding the module resolution magic, there is [a plugin](https://github.com/tleunen/babel-plugin-module-resolver) that might be worth investigating for this purpose, but I haven't tried it out. It essentially allows you to create an alias for the imports.

Reply:
> Regarding the module resolution magic, there is a plugin that might be worth investigating for this purpose, but I haven't tried it out. It essentially allows you to create an alias for the imports.

I've used that plugin in every React native project I've worked on, it's very useful. Also very [easy to set up](https://dev.to/karanpratapsingh/cleaning-up-imports-using-module-resolver-in-react-native-58g8) (usually).